seqr/views/apis/dataset_api_tests.py

```diff
@@ -118,6 +122,13 @@ def test_add_variants_dataset(self, mock_random):
         self.assertEqual(len(updated_sample_models), 3)
         self.assertSetEqual({INDEX_NAME}, {sample.elasticsearch_index for sample in updated_sample_models})
 
+        mock_send_email.assert_not_called()
+        mock_send_slack.assert_called_with(
+            'seqr-data-loading',
+            '1 new sample(s) are loaded in https://seqr.broadinstitute.org/project/{guid}/project_page\n            ```[\'NA19678_1\']```\n            '.format(
```

Comment:
its important to test that we are only notifying on the new samples, but it looks like the test input data only includes new samples to begin with. we need to update that

Reply:
I will add tests for slack messages after adding the two additive datasets. The notice should include the new samples only.